src/Backend/Modules/Pages/Domain/Page/PageRepository.php

```diff
@@ -396,7 +396,7 @@ public function findOneByParentsAndUrlAndStatusAndLanguageExcludingId(
             ->getOneOrNullResult();
     }
 
-    private function buildGetQuery(int $pageId, int $revisionId, string $language): QueryBuilder
```

Comment:
wouldn't it make more sense to use `private function buildGetQuery(int $pageId, int $revisionId = null, string $language = null): QueryBuilder`

Reply:
It was changed in ea494073e69a85512a611ca895f849605e5ff1b9